SyntheticGeometry/ProjectiveSpace.lagda.md

```diff
@@ -58,15 +59,15 @@ open import SyntheticGeometry.SQC.Consequences k k-local k-sqc
 module _ (n : ℕ) where
   private
     module k = CommRingStr (snd k)
-    module 𝔸ⁿ⁺¹ = LeftModuleStr (snd (FinVecLeftModule (CommRing→Ring k) {n = n + 1}))
+    module 𝔸ⁿ⁺¹ = LeftModuleStr (snd (FinVecLeftModule (CommRing→Ring k) {n = 1 + n}))
   open k hiding (_+_)
   open 𝔸ⁿ⁺¹ hiding (_+_)
   open Units k
 
-  𝔸ⁿ⁺¹ = FinVec ⟨ k ⟩ (n + 1)
+  𝔸ⁿ⁺¹ = FinVec ⟨ k ⟩ (1 + n)
```

Comment:
Do we really want to keep the name `𝔸ⁿ⁺¹` if we actually use `1 + n` now?

Reply:
If we want to keep the name, we should add a comment about `n + 1` vs `1 + n` here.

Reply:
I decided for a comment.